api/src/main/java/com/google/sps/Email.java

```diff
@@ -46,4 +43,17 @@ public String getSubject(){
   public String getEmail(){
     return player.getEmail();
   }
+
+  static Email populateEmail(String gameID, String playerEmail, String playerName, EmailType emailType)
+    throws RuntimeException {
+
+    switch(emailType) {
+      case END:
+        return endGameEmail(gameID, new User(playerEmail, playerName));
+      default:
+        System.out.println("Invalid email type");
```

Comment:
Don't really understand the point of emailType parameter if it can only call one email type. This is essentially calling endGameEmail with extra steps